app/helpers/shopping_lists_helper.rb

```diff
@@ -1,2 +1,7 @@
 module ShoppingListsHelper
+
+  def shopping_list_statuses
+    @shopping_list_statuses = ShoppingList.statuses.keys
```

Comment:
We can add small optimization here. As `ShoppingList.statuses.keys` is something which won't change, there is no need to do this assignment everything when `shopping_list_statuses` is called. Ruby has useful operator `||=`  which can be used for memoization. This operator will only assign value to the variable if its previous value was nil.
So it can be used instead of `=` here `@shopping_list_statuses ||=`